.github/workflows/kube-linter.yaml

```diff
@@ -0,0 +1,54 @@
+name: Check Kubernetes YAMLs with kube-linter
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'config/default/**.ya?ml'
+      - 'config/crd/**.ya?ml'
+      - 'config/rbac/**.ya?ml'
+      - 'config/manager/**.ya?ml'
+      - 'config/monitoring/prometheus/**.ya?ml'
+
+jobs:
+  kube-linter:
+    name: Kube linter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create ../kube-linter/ for deployment yaml files
+        shell: bash
+        run: mkdir -p ../kube-linter/
+
+      - name: Generate Build Service operator deployment configuration
+        shell: bash
+        run: kustomize build config/default/ >  ../kube-linter/build-service.yaml
```

Comment:
Do we have to setup the kustomize to make it available in the env like it is done [here](https://github.com/redhat-appstudio/infra-deployments/blob/main/.github/workflows/kube-linter.yaml#L20) or will it be available with ubuntu-latest image?